tests/SideBySide/ConnectionPool.cs

```diff
@@ -205,35 +205,6 @@ public void LeakReaders()
 			}
 		}
 
-		[Theory]
-		[InlineData(2u, 3u, true)]
```

Comment:
Removing this specific test makes `LeakReaders` fail; apparently there's some interference between the two tests 😞